### PR TITLE
Add support for .NET 6

### PR DESCRIPTION
**Pull request by azabbasi**

### CODE REVIEW

File: vsts/test-release-nuget.yaml [hunk 1]
```diff
@@ -13,6 +13,8 @@ jobs:
       matrix:
         .Net 5.0:
           FRAMEWORK: net5.0
+        .Net 6.0:
```
Comment (abhipsaMisra) [inline vsts/test-release-nuget.yaml:16]: do we plan to release a .net 6.0 dll as well?

Comment (azabbasi) [reply vsts/test-release-nuget.yaml:16]: good question, I don't think so, there is nothing new there, net5.0 dll should suffice. I shouldn't add it to the test-release matrix then right?

**Review by timtay-microsoft — APPROVED** [review]

**Pull request merged**